leaflet-geoman.d.ts

```diff
@@ -1,340 +1,779 @@
+            };
+
+            actions?: {
+                finish?: string;
+                cancel?: string;
+                removeLastVertex?: string;
+            };
+
+            buttonTitles?: {
+                drawMarkerButton?: string;
+                drawPolyButton?: string;
+                drawLineButton?: string;
+                drawCircleButton?: string;
+                drawRectButton?: string;
+                editButton?: string;
+                dragButton?: string;
+                cutButton?: string;
+                deleteButton?: string;
+                drawCircleMarkerButton?: string;
+            };
+        }
+
+        type ACTION_NAMES = 'cancel' | 'removeLastVertex' | 'finish' | 'finishMode';
+
+        class Action {
+            text: string;
+            onClick: (e: any) => void;
+        }
+
+        interface PMMapToolbar {
+            /** Pass an array of button names to reorder the buttons in the Toolbar. */
+            changeControlOrder(order: 'drawCircle' | 'drawRectangle' | 'removalMode' | 'editMode'[]): void;
+
+            /** Receive the current order with. */
+            getControlOrder(): 'drawCircle' | 'drawRectangle' | 'removalMode' | 'editMode'[];
+
+            /** The position of a block (draw, edit, custom, options⭐) in the Toolbar can be changed. If not set, the value from position of the Toolbar is taken. */
+            setBlockPosition(block: 'draw' | 'edit' | 'custom' | 'options', position: ControlPosition): void;
+
+            /** Returns a Object with the positions for all blocks */
+            getBlockPositions(): BlockPositions;
+
+            /** To add a custom Control to the Toolbar */
+            createCustomControl(options: CustomControlOptions): void;
+
+            /** Creates a copy of a draw Control. Returns the drawInstance and the control. */
+            copyDrawControl(copyInstance: string, options?: CustomControlOptions): void;
+
+            /** Change the actions of an existing button. */
+            changeActionsOfControl(name: string, actions: (ACTION_NAMES | Action)[]): void;
+        }
+
+        interface Button {
+            /** actions */
+            actions: (ACTION_NAMES | Action)[];
+
+            /** Function fired after clicking the control. */
+            afterClick: () => void;
+
+            /** CSS class with the Icon. */
+            className: string;
+
+            /**  */
+            disableOtherButtons: boolean;
+
+            doToggle: boolean;
+
+            jsClass: string;
+
+            /** Function fired when clicking the control. */
+            onClick: () => void;
+
+            position: L.ControlPosition;
+
+            title: string;
+
+            toggleStatus: boolean;
+        }
+
+        interface CustomControlOptions {
+            /** Name of the control */
+            name: string;
+
+            /** block of the control. 'options' is ⭐ only. */
+            block?: 'draw' | 'edit' | 'custom' | 'options';
```

Comment:
"block: is it possible to use BlockPositions? Else please add options ⭐"

this needs to be set to the string correct? `BlockPosition` is an object with properties